spec/rails_helper.rb

```diff
@@ -12,14 +12,27 @@
 include Warden::Test::Helpers
 Warden.test_mode!
 
+options = {
+  js_errors: false,
```

Comment:
Do you know why we have this on?

Reply:
i do not, i just copied over the same vcr configurations for the website.